_components/accordions.md

```diff
@@ -4,9 +4,153 @@ type: component
 title: Accordions
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Accordions_UI_v1.png">
+<h3>Bordered Accordion</h3>
+
+<div class="code-sample">
+  <ul id="accordion-menu">
```

Comment:
Accordions are tricky. They increase the number of clicks necessary to traverse a page's content. They make it more difficult to compare content on two pages. They're generally a bit of a usability minefield. See http://www.nngroup.com/articles/accordions-complex-content/.

That said, if a developer makes a conscious decision to use one accordion, I can't think of a reason why it would be bad to use multiple ones.

I'd love to hear @carodew's thoughts.